types/cron_interval.pp

```diff
@@ -0,0 +1,10 @@
+# The type of job
```

Comment:
can you add puppet-strings documentation for the two new types?
```
# @summary ...
```

should be sufficient (same for the other new type

Reply:
Outch!

Reply:
(for the comment I mean… copy/paste/forget-to-remove-or-adjust)